tools/install_synapse.sh

```diff
@@ -22,9 +22,15 @@ if [[ ! -d ${DESTDIR} ]]; then
     fi
 fi
 
+# this function sets the modification time of file to time of last commit
+gitmtime(){ local f;for f;do touch -d @0`git log --pretty=%at -n1 -- "$f"` "$f"; done; }
```

Comment:
what is the purpose of the loop? `for f` is not iterating over anything.

Reply:
Not sure, just pasted this code from the internetz, but this code was removed anyway in current master for a commit-versioned naming scheme over these mtime-based updates.